firebase-functions/src/androidTest/java/com/google/firebase/functions/CallTest.java

```diff
@@ -97,16 +85,15 @@ public void testToken() throws InterruptedException, ExecutionException {
     FirebaseFunctions functions =
         new FirebaseFunctions(
             app.getApplicationContext(),
-            "functions-integration-test",
+            app.getOptions().getProjectId(),
             "us-central1",
-            new ContextProvider() {
-              @Override
-              public Task<HttpsCallableContext> getContext() {
-                HttpsCallableContext context = new HttpsCallableContext("token", null);
-                return Tasks.forResult(context);
-              }
+            () -> {
+              HttpsCallableContext context = new HttpsCallableContext("token", null);
+              return Tasks.forResult(context);
             });
-    useTestURL(functions);
+    // 'Authorization' is a special header that does not reach the function if it does not contain a
+    // valid token.
+    functions.replaceAuthorizationHeaderForTest("Test-Authorization");
```

Comment:
Are you sure? The test functions aren't implemented as callables, so I don't think the token gets checked. We haven't had to do this on iOS or JavaScript, so I'd really like to not have to add it here.

Reply:
You're right, I don't know why it was failing for me on Friday, now it is passing. Removed.